docker-compose.yml

```diff
@@ -116,3 +127,6 @@ services:
     entrypoint: java -jar tika-server-1.23.jar -h 0.0.0.0 -spawnChild
     ports:
       - "9998:9998"
+
+volumes:
+  opensearch-data1:
```

Comment:
Should this be an empty key?

Reply:
Yes - the mapping is declared in the `opensearch-node1` definition:
```- opensearch-data1:/usr/share/opensearch/data```

The volumes here is referring back to that when docker compose builds the... build?

Reply:
It looks weird but it is okay. It is really:

```
volumes:
  opensearch-data1: {}
```

https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes

Reply:
Ok sounds good!